pkg/caimake/cmd/clean.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/caicloud/nirvana/cli"
```

Comment:
This package is gone, should we use cobra directly?

Reply:
I lock `nirvana` with version `v0.1.0-alpha.0` where `cli` pkg is still alive.
I‘d like to replace it util `mamba` rebirth.